github/ci/prow-deploy/kustom/base/configs/current/plugins/plugins.yaml

```diff
@@ -546,6 +546,10 @@ external_plugins:
     endpoint: http://referee:9900
     events:
       - issue_comment
+  - name: github-jira-proxy
```

Comment:
This should be removed until the code matches the setup for a proper external plugin.